.github/workflows/api-endpoint.yml

```diff
@@ -0,0 +1,54 @@
+name: API tests
+
+on: [push]
+
+env:
+  RUST_BACKTRACE: full
+
+jobs:
+  api:
+    name: "The API endpoints"
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Install Packages"
+        run: sudo apt-get install -y libmpfr6 libmpfr-dev
+      - name: "Install Racket"
+        uses: Bogdanp/setup-racket@v1.8
+        with:
+          version: 8.5
+      - name: Install Rust compiler
+        uses: actions-rs/toolchain@v1
+        with:
+            toolchain: stable
+            default: true
+            override: true
+            components: rustfmt, clippy
+      - uses: actions/checkout@master
+      - name: "Install dependencies"
+        run: make install
+
+      - name: "Start server"
+        run: racket src/herbie.rkt web --port 8000 &
+      
+      - name: "Pause 4 seconds"
+        run: sleep 4s
```

Comment:
Kind of ugly. I get why it's there—I'm just trying to think through if this will be flaky. Happy to merge without changing but would appreciate a comment about what our backup is.

Reply:
I agree it's ugly. An alternative is to just wait for an open port with netcat + sleep (and do performance testing elsewhere to catch server hanging on startup). I'll go ahead and change this before merge unless you indicate I should leave it alone. https://stackoverflow.com/questions/27599839/how-to-wait-for-an-open-port-with-netcat